server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessor.java

```diff
@@ -694,18 +703,26 @@ public void flushOneMemTable() {
         MemTableFlushTask flushTask = new MemTableFlushTask(memTableToFlush, writer, storageGroupName);
         flushTask.syncFlushMemTable();
       } catch (Exception e) {
-        logger.error("{}: {} meet error when flushing a memtable, change system mode to read-only",
-            storageGroupName, tsFileResource.getTsFile().getName(), e);
-        IoTDBDescriptor.getInstance().getConfig().setReadOnly(true);
-        try {
-          logger.error("{}: {} IOTask meets error, truncate the corrupted data", storageGroupName,
-              tsFileResource.getTsFile().getName(), e);
-          writer.reset();
-        } catch (IOException e1) {
-          logger.error("{}: {} Truncate corrupted data meets error", storageGroupName,
-              tsFileResource.getTsFile().getName(), e1);
+        if (writer == null) {
+          logger.info("{}: {} is closed during flush, abandon flush task",
```

Comment:
is this just an "info"?  Maybe a WARN or ERROR?

Reply:
It is normal, as currently only loading a file that overlaps it may close it during flush.